charts/jenkins/README.md

```diff
@@ -79,343 +116,48 @@ So think of the list below more as a general guideline of what should be done.
 - prepare your values file for the update e.g. add additional configuration as code setting that you need.
   The export taken from above might be a good starting point for this.
   In addition the [demos](https://github.com/jenkinsci/configuration-as-code-plugin/tree/master/demos) from the plugin itself are quite useful.
-- test drive those setting on a separate installation
+- Test drive those setting on a separate installation
 - Put Jenkins to Quiet Down mode so that it does not accept new jobs
   `<JENKINS_URL>/quietDown`
-- change permissions of all files and folders to the new user and group id
+- Change permissions of all files and folders to the new user and group id:
 
-  ```shell script
+  ```console
   kubectl exec -it <jenkins_pod> -c jenkins /bin/bash
   chown -R 1000:1000 /var/jenkins_home
   ```
+
 - Update Jenkins
 
-### 1.0.0
+### To 1.0.0
 
 Breaking changes:
 
-- values have been renamed to follow helm chart best practices for naming conventions so
-  that all variables start with a lowercase letter and words are separated with camelcase
-  https://helm.sh/docs/chart_best_practices/#naming-conventions
-- all resources are now using recommended standard labels
-  https://helm.sh/docs/chart_best_practices/#standard-labels
+- Values have been renamed to follow [helm recommended naming conventions](https://helm.sh/docs/chart_best_practices/#naming-conventions) so that all variables start with a lowercase letter and words are separated with camelcase
+- All resources are now using [helm recommended standard labels](https://helm.sh/docs/chart_best_practices/#standard-labels)
 
 As a result of the label changes also the selectors of the deployment have been updated.
 Those are immutable so trying an updated will cause an error like:
 
-```
+```console
 Error: Deployment.apps "jenkins" is invalid: spec.selector: Invalid value: v1.LabelSelector{MatchLabels:map[string]string{"app.kubernetes.io/component":"jenkins-master", "app.kubernetes.io/instance":"jenkins"}, MatchExpressions:[]v1.LabelSelectorRequirement(nil)}: field is immutable
 ```
 
-In order to upgrade, delete the Jenkins Deployment before upgrading:
+In order to upgrade, [uninstall](#uninstall-chart) the Jenkins Deployment before upgrading:
 
-```
-kubectl delete deploy jenkins
-```
+## Configuration
 
+See [Customizing the Chart Before Installing](https://helm.sh/docs/intro/using_helm/#customizing-the-chart-before-installing). To see all configurable options with detailed comments, visit the chart's [values.yaml](./values.yaml), or run these configuration commands:
 
-## Configuration
+```console
+# Helm 2
+$ helm inspect values jenkinsci/jenkins
 
-The following tables list the configurable parameters of the Jenkins chart and their default values.
-
-### Jenkins Master
```

Comment:
These tables might have had some value as user documentation for those who have no immediate access to Helm, but I'd guess maintaining them is difficult

Reply:
@wmcdona89 added them back in a separate file.